### PR TITLE
Click Away Handler

### PR DESCRIPTION
**Pull request by harismahmood89**
- add transparent overlay
- click away handler function
- clean up open/close function calls for cart

### CODE REVIEW

**Comment by harismahmood89** [comment]
Please review @xto @richgilbank

File: examples/cart/index.js [hunk 1]
```diff
@@ -158,13 +159,29 @@ $(function() {
...
   ============================================================ */
   function closeCart() {
+    $('.cart').removeClass('js-active');
+    $('.overlay').removeClass('js-active');
+  }
+
+  /* Close Cart Handler
+  ============================================================ */
+  function closeCartHandler() {
     $('.cart .btn--close').click(function () {
```
Comment (minasmart) [inline examples/cart/index.js]: `.click(closeCart)`

**Comment by minasmart** [comment]
One teeny tiny thing. Looks good otherwise 👍

**Comment by minasmart** [comment]
🐑

**Comment by richgilbank** [comment]
Not clear what's included specifically for this PR vs https://github.com/Shopify/js-buy-sdk/pull/82 - anything here that's not included in that one?

**Comment by harismahmood89** [comment]
a branched off of here for #82 so this is no longer needed

**Pull request closed without merging**